paymentsheet/src/main/java/com/stripe/android/paymentsheet/address/AddressFieldElementRepository.kt

```diff
@@ -50,6 +50,13 @@ internal class AddressFieldElementRepository @Inject internal constructor(
         countryFieldMap[countryCode] = listElements
     }
 
+    internal fun getCountriesAllowedForKlarna(currencyCode: String?): Set<String> {
```

Comment:
`AddressFieldElementRepository` seems a generic util that could get return a list of countries for all types of payments, is it possible to expose a generic method with "klarnaCurrencyMap.json" passed in as a param, instead of `Klarna` specifc method here?